Sources/Core/Strings/LocalizedStringHelper.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+//
+// You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+// copy, modify, and distribute this software in source code or binary form for use
+// in connection with the web services and APIs provided by Facebook.
+//
+// As with any software that integrates with the Facebook platform, your use of
+// this software is subject to the Facebook Developer Principles and Policies
+// [http://developers.facebook.com/policy/]. This copyright notice shall be
+// included in all copies or substantial portions of the software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+enum LocalizedStringHelper {
```

Comment:
You're returning Bundle.main twice. Probably best to do something like this:

```suggestion
enum LocalizedStringHelper {
  static var localizedStringsBundle: Bundle = {
    guard let path = Bundle(for: FacebookCoreApplicationDelegate.self)
      .path(forResource: "FacebookSDKStrings", ofType: "bundle"),
      let bundle = Bundle(path: path) else {
      return Bundle.main
    }

    return bundle
  }()

```

Reply:
Fixing. Good call. 💯